WEB_SERVICE/WEB/_MEDIUM/GRAPHQL_OBJECT_LIMIT_OVERRIDING/description.md

```diff
@@ -0,0 +1,15 @@
+Object Limit Overriding in GraphQL refers to a vulnerability where an attacker can manipulate the request to exceed the intended limits of an API, leading to potential denial-of-service (DoS) attacks or excessive resource consumption. This is particularly concerning in GraphQL, where complex queries can result in extensive data retrieval and processing if not properly restricted.
+
+### Potential Risks:
+
+- **Denial of Service (DoS):** Attackers may send overly complex queries that consume excessive server resources, causing performance degradation or outages.
+- **Resource Exhaustion:** Large queries or mutations can overload the server’s memory and processing capabilities.
+- **Increased Costs:** For cloud-based services, complex queries may result in unexpected costs due to high resource usage.
+
+### How to Check:
```

Comment:
Avoid the subsection, also missing code source samples. You can also remove how to check since that is our job.